src/components/materialization/create/Savebutton.tsx

```diff
@@ -0,0 +1,113 @@
+import { Button } from '@mui/material';
+import { EditorStoreState } from 'components/editor/Store';
+import { buttonSx } from 'components/shared/Entity/Header';
+import { useClient } from 'hooks/supabase-swr';
+import { DraftSpecQuery } from 'hooks/useDraftSpecs';
+import { useRouteStore } from 'hooks/useRouteStore';
+import { useZustandStore } from 'hooks/useZustand';
+import { FormattedMessage } from 'react-intl';
+import { TABLES } from 'services/supabase';
+import {
+    entityCreateStoreSelectors,
+    formInProgress,
+    FormStatus,
+} from 'stores/Create';
+
+interface Props {
+    disabled: boolean;
+    formId: string;
+    onFailure: Function;
+    subscription: Function;
+}
+
+function MaterializeSaveButton({
```

Comment:
Same as capture - but materialize. Eventually these two entities can share some more functionality in their save/test components.